File: Day-01 ( Age Estimate From Days Lived ) /CPP/Solution.cpp
#include <iostream>
using namespace std;

int main() {
    cout << "=== Age Estimator ===" << endl;

    int days;
    cout << "Enter the number of days you have lived: ";
    cin >> days;

    // Calculate years
    int years = days / 365;
    int remainingDays = days % 365;

    // Calculate months
    int months = remainingDays / 30;
    int finalDays = remainingDays % 30;

    cout << endl;
    cout << "Your estimated age is:" << endl;
    cout << years << " years, "
         << months << " months, "
         << finalDays << " days" << endl;

    return 0;
}

File: Day-01 ( Age Estimate From Days Lived ) /Python/Solution.py
print("=== Age Estimator ===")

days = int(input("Enter the number of days you have lived: "))

# Calculate years
years = days // 365
remaining_days = days % 365

# Calculate months
months = remaining_days // 30
final_days = remaining_days % 30

print()
print("Your estimated age is:")
print(f"{years} years, {months} months, {final_days} days")
